lib/utils/cloud.ts

```diff
@@ -150,10 +148,17 @@ export async function downloadOSImage(
 	const manager = await import('balena-image-manager');
 	const stream = await manager.get(deviceType, OSVersion);
 
+	const displayVersion = await new Promise((resolve, reject) => {
+		stream.on('error', reject);
+		stream.on('version', resolve);
```

Comment:
This relies on a new `'version'` event introduced in balena-io-modules/balena-image-manager/pull/54